client/src/components/relatedProducts/RelatedProducts.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import $ from 'jquery';
+import ProductCard from './ProductCard.jsx';
+import YourOutfit from './YourOutfit.jsx';
+import Modal from './Modal.jsx';
+
+class RelatedProducts extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      // productId: '28212',
+      // productInfo: [],
+      // productDetailInfo: []
+    }
+  }
```

Comment:
if RelatedProducts ends up not having any class, perhaps you could change it into a functional component?

Reply:
Totally.  I'm still up in the air about it.  This went back and forth from functional to class like 3 times already. I'm plan on leaving it like this until I get closer to finished and converting then if I need to.